go/libkb/base58.go

```diff
@@ -14,10 +14,12 @@ var alphabetMap map[byte]uint8
 
 func getAlphabetMap() map[byte]uint8 {
 	if alphabetMap == nil {
-		alphabetMap = make(map[byte]uint8)
+		newMap := make(map[byte]uint8)
 		for i, c := range []byte(alphabet) {
-			alphabetMap[c] = uint8(i)
+			newMap[c] = uint8(i)
 		}
+		// We do this atomically to avoid needing to take a lock.
```

Comment:
I mainly agree, but I think you'll still get a weird pointer write when they both try to write the new `alphabetMap`. I think `once.Do` is the preference technique...